src/bio/bio_xstream.c

```diff
@@ -25,7 +25,8 @@
 /* These Macros should be turned into DAOS configuration in the future */
 #define DAOS_MSG_RING_SZ	4096
 /* SPDK blob parameters */
-#define DAOS_BS_CLUSTER_SZ	(1ULL << 30)	/* 1GB */
+#define DAOS_BS_DATA_CLUSTER_SZ	(1ULL << 30)	/* 1GB */
+#define DAOS_BS_META_CLUSTER_SZ	(1ULL << 25)	/* 32MB */
```

Comment:
The cluster size is per blobstore, so I think we can't use mixed cluster size when meta blob and data blob resides in same device/blobstore.

Reply:
if meta blob and data blob resides in the same device, we will use DATA_CLUSTER_SZ

Reply:
That'll be too large for sysdb and RDB. I think we should always use 32MB cluster size to keep it simple.